src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/util/DocFinder.java

```diff
@@ -134,32 +114,35 @@ public Input(Utils utils, Element element, InheritableTaglet taglet) {
         }
 
         public Input(Utils utils, Element element) {
-            if (element == null)
-                throw new NullPointerException();
-            this.element = element;
+            this.element = Objects.requireNonNull(element);
             this.utils = utils;
-            this.docTreeInfo = new DocTreeInfo();
+            this.docTreeInfo = new DocTreeInfo(null, null);
         }
 
-        public Input(Utils utils, Element element, boolean isFirstSentence) {
-            this(utils, element);
+        public Input(Utils utils,
+                     Element element,
+                     InheritableTaglet taglet,
+                     DocTreeInfo dtInfo,
+                     boolean isFirstSentence,
+                     boolean isInheritDocTag) {
+            this.utils = utils;
+            this.element = Objects.requireNonNull(element);
+            this.taglet = taglet;
             this.isFirstSentence = isFirstSentence;
+            this.isInheritDocTag = isInheritDocTag;
+            this.docTreeInfo = dtInfo;
         }
 
-        public Input copy(Utils utils) {
-            if (this.element == null) {
-                throw new NullPointerException();
-            }
-            Input clone = new Input(utils, this.element, this.taglet, this.docTreeInfo,
-                    this.isFirstSentence, this.isInheritDocTag);
-            clone.tagId = this.tagId;
-            clone.isTypeVariableParamTag = this.isTypeVariableParamTag;
-            return clone;
+        private Input copy(Utils utils) {
```

Comment:
Why do you need to pass in `utils` -- you can just use the value in the `Input` object being copied.

Reply:
I don't need to pass `utils`; good catch.